pkgs/by-name/co/collectl/test.nix

```
@@ -0,0 +1,44 @@
+{
+  runCommand,
+  collectl,
+  coreutils,
+}:
+
+runCommand "collectl-test"
+  {
+    nativeBuildInputs = [
+      collectl
+      coreutils
+    ];
+    meta.timeout = 60;
+  }
+  ''
+    # Test basic functionality - limit to 5 seconds to avoid hanging
+    timeout 5s ${collectl}/bin/collectl -c1 >/dev/null || true
+
+    # Test that explicit config file option still works with original config
+    timeout 5s ${collectl}/bin/collectl --config ${collectl}/etc/collectl.conf -c1 >/dev/null || true
```

```suggestion
    timeout 5s collectl --config ${collectl}/etc/collectl.conf -c1 >/dev/null || true
```
`collectl` is part of `nativeBuildInputs`.